src/app/services/relationship.service.ts

```diff
@@ -12,6 +12,8 @@ export class RelationshipService {
   constructor(private http: HttpClient) { }
 
   getRelatedArtifacts(resource: string, uid: string, relationship: string): Observable<any> {
-    return this.http.get(`${this.apiUrl}relationships/${resource}/${uid}/${relationship}`);
+    const userInput = encodeURIComponent(`${uid}`);
```

Comment:
Although `${uid}` works, it's easier to read if you just use the `uid` variable directly as in `const userInput = encodeURIComponent(uid);`.